scripts/lib/octavia/build_certificates.sh

```diff
@@ -0,0 +1,187 @@
+#!/bin/bash
+# requires openssl.conf inside this directory
```

Comment:
Not strictly true anymore, as it now will drop a default one if it isn't there. I'll update the comment on the next commit.